src/defs.asm

```diff
@@ -703,6 +703,16 @@ Enemy_BossMushroom = $7F
 
 ; ---------------------------------------------------------------------------
 
+LevelDirection_Horizontal = 1
+LevelDirection_Vertical = 0
+
+LevelMusic_Overworld = 0
+LevelMusic_Underground = 1
+LevelMusic_Boss = 2
+LevelMusic_Wart = 3
```

Comment:
this is mostly to help keep the `levelHeader` macro readable

since the other parameters didn't seem to make as much sense as enums, i decided it wasn't worth making the lines extra long

Reply:
This seems fine to me. I was playing around with this on my own and came to the same conclusions, certainly for the 5-bit header field at the very least.